### PR TITLE
Use test write key for non-release builds

### PR DESCRIPTION
**Pull request by guillaumerose**

### CODE REVIEW

File: pkg/crc/segment/segment.go [hunk 1]
```diff
@@ -58,6 +58,10 @@ func (c *Client) Close() error {
 }
 
 func (c *Client) Upload(action string, duration time.Duration, err error) error {
+	if version.IsOkdBuild() {
+		return nil
```
Comment (praveenkumar) [inline pkg/crc/segment/segment.go:62]: Are we sure about this (cc @sspeiche), Our motive is crc usage collection, doesn't matter if it is okd or ocp right?

Comment (guillaumerose) [reply pkg/crc/segment/segment.go:62]: Some arguments:
* Community doesn't have access to Segment. They can't look at statistics. 
* The ID we used is also something Red Hat related. 
* I would expect community builds to be free from all sort of external services.

Comment (sspeiche) [reply pkg/crc/segment/segment.go:62]: Agree we should try to keep out off community bits, though would be interesting to know how many OKD instances there are if we can...I think we just need to be transparent we are doing it and allow for opt-out.

Comment (praveenkumar) [reply pkg/crc/segment/segment.go:62]: @sspeiche As of now by default it is opt out, it is upto the user to enable it or not.

Comment (guillaumerose) [reply pkg/crc/segment/segment.go]: OK, I removed this from the PR. OKD will send events if turned on.

File: Makefile [hunk 2]
```diff
@@ -172,6 +172,7 @@ gen_release_info:
 	@sed -i s/@OPENSHIFT_VERSION@/\"$(BUNDLE_VERSION)\"/ $(RELEASE_INFO)
 
 .PHONY: release
+release: LDFLAGS := $(LDFLAGS) -X $(REPOPATH)/pkg/crc/segment.WriteKey=cvpHsNcmGCJqVzf6YxrSnVlwFSAZaYtp
```
Comment (praveenkumar) [inline Makefile:175]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**